server/protocols/jmap/src/main/java/org/apache/james/jmap/methods/SetMailboxesCreationProcessor.java

```diff
@@ -19,6 +19,8 @@
 
```

Comment:
I don't agree with this commit : you are escaping strings everywhere you find it relevent.
Needless to say that it opens the door to missing the right call when the app will grow.
Nothing will prevent me from missing this call.
I suggest to use the type system to make sure the escape/unescape is always done, WDYT ?